reporters/src/main/java/com/nextdoor/bender/monitoring/cw_embedded_metrics/CloudWatchMetricObject.java

```diff
@@ -0,0 +1,49 @@
+package com.nextdoor.bender.monitoring.cw_embedded_metrics;
```

Comment:
See the style guide [1] re naming of packages.

[1] https://google.github.io/styleguide/javaguide.html#s5.2.1-package-names

Reply:
Changed to embedded.metrics